streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -360,7 +360,8 @@ public void suspend() {
      *   - commit offsets
      * </pre>
      */
-    private void suspend(final boolean clean) {
+    // visible for testing
+    void suspendImpl(final boolean clean) {
```

Comment:
why `suspendImpl`? That is a pretty horrible name for a method. Similarly `commitImpl`